src/environments/environment.tns.ts

```diff
@@ -0,0 +1,27 @@
+import { environment as devEnvironment } from './environment.dev';
+import { environment as prodEnvironment } from './environment.prod';
+import { environment as stageEnvironment } from './environment.stage';
+
+export const environment = (() => {
+    let envVars;
+
+    if (Object.prototype.hasOwnProperty.call(process.env, 'environment') &&
```

Comment:
Is not enough to have this condition right here ```process.env.environment```?

Reply:
We double check if webpack have right configuration.